Sources/Vortex/Views/VortexView.swift

```diff
@@ -55,6 +56,32 @@ public struct VortexView<Symbols>: View where Symbols: View {
         self.symbols = symbols()
     }
 
+    /// Creates a new VortexView from a pre-configured particle system, along with any required  SwiftUI
+    /// views needed to render particles. Sensible defaults will be used if no parameters are passed.
+    /// - Parameters:
+    ///   - settings: A vortexSettings struct that should be used to generate a particle system. 
+    ///               Typically this will be set using a preset static struct, e.g. `.fire`. Defaults to a simple system.
+    ///   - targetFrameRate: The ideal frame rate for updating particles. Defaults to 60 if not specified. (use 120 on Pro model iPhones/iPads )
+    ///   - symbols:  A closure that should return a tagged group of SwiftUI views to use as particles. 
+    ///               If not specified, a default group of three views tagged with `.circle`,`.confetti` and `.sparkle` will be used.
+    public init(
+        settings: VortexSystem.Settings = .init(),
```

Comment:
I think we should probably use `_ settings` so that we get `VortexView(.confetti)`. I'd be happy to rename the older one and mark it deprecated if it's causing confusion, but I want the newer, correct option to be the simplest.

Reply:
agreed